packages/blade/src/components/Amount/Amount.stories.tsx

```diff
@@ -217,3 +223,98 @@ StrikeThrough.args = {
   isStrikethrough: true,
 };
 StrikeThrough.storyName = 'Strike Through';
+
+// TODO: Replace below with i18nify getDefaultLocales
+const localeList = [
+  {
+    country: 'India',
+    locale: 'en-IN',
+  },
+  {
+    country: 'USA',
+    locale: 'en-US',
+  },
+  {
+    country: 'Malaysia',
+    locale: 'ms-MY',
+  },
+  {
+    country: 'France',
+    locale: 'fr-FR',
+  },
+  {
+    country: 'Germany',
+    locale: 'de-DE',
+  },
+];
+
+const I18nAmountWrapper = (args: AmountProps) => {
+  const { setI18nState } = useI18nContext();
+  const [currency, setCurrency] = useState('INR');
+
+  return (
+    <>
+      <AmountComponent {...args} currency={currency as AmountProps['currency']} />
+      <Divider marginY="spacing.4" marginTop="spacing.8" />
+      <Dropdown selectionType="single">
+        <SelectInput label="Select locale" />
+        <DropdownOverlay>
+          <ActionList>
+            {localeList.map((item) => (
+              <ActionListItem
+                key={item.locale}
+                title={`${item.country}(${item.locale})`}
+                value={item.locale}
+                onClick={({ name }) => {
+                  setI18nState({ locale: name });
+                }}
+              />
+            ))}
+          </ActionList>
+        </DropdownOverlay>
+      </Dropdown>
+      <Divider marginY="spacing.4" />
+      <Dropdown selectionType="single">
+        <SelectInput label="Select currency" />
+        <DropdownOverlay>
+          <ActionList>
+            {Object.keys(getCurrencyList()).map((value) => (
+              <ActionListItem
+                key={value}
+                title={value}
+                value={value}
+                onClick={({ name }) => {
+                  setCurrency(name);
+                }}
+              />
+            ))}
+          </ActionList>
+        </DropdownOverlay>
+      </Dropdown>
+    </>
+  );
+};
+
+const I18nAmountTemplate: StoryFn<typeof AmountComponent> = (args) => {
+  return (
+    <I18nProvider>
+      <BaseBox justifyContent="flex-start" maxHeight="300px" overflowY="auto">
```

Comment:
I think here you can set a minHeight to ensure the dropdown gets sufficient space to show options.

Reply:
It looks a bit odd but does the job for now. @snitin315 , have updated.
<img width="275" alt="Screenshot 2024-02-06 at 10 13 47 PM" src="https://github.com/razorpay/blade/assets/21965506/ac52da14-32bb-4eb8-9415-e233e11347bd">